R/drive_mv.R

```diff
@@ -87,11 +93,8 @@ drive_mv <- function(file, path = NULL, name = NULL, verbose = TRUE) {
     }
   }
 
-  request <- generate_request(
```

Comment:
Showing the addition or deletion of a parent would make a great example for `drive_update()`, actually. It's a legit example, plus people might need to do this in multiple parent situations and we provide no direct support for that. You should also rip out my un-exported `drive_add_parent()` function and replace its usage in test file setup with a call to `drive_update()`.